pkg/skuba/actions/node/bootstrap/bootstrap.go

```diff
@@ -68,6 +68,8 @@ func Bootstrap(bootstrapConfiguration deployments.BootstrapConfiguration, target
 		return errors.Wrap(err, "unable to add target information to init configuration")
 	}
 
+	setConfigurationExtraArgs(initConfiguration)
```

Comment:
I think all the changes in `bootstrap.go` can be removed in this PR. Changing the `init` template will allow the user to easily change the `extraArgs` between `cluster init` and `node bootstrap`.

Setting defaults on `bootstrap` to empty maps has no effect.

Reply:
Ok, you are right, I have checked it and it works as you expected